build.cake

```diff
@@ -202,7 +203,8 @@ Task("Test45")
     {
         var runtime = "net45";
         var dir = BIN_DIR + runtime + "/";
-        RunNUnitTests(dir, FRAMEWORK_TESTS, runtime, ref ErrorDetail);
+        RunTest(dir + EXECUTABLE_NUNITLITE_TEST_RUNNER_EXE, dir, FRAMEWORK_TESTS, dir + "nunit.framework.tests.xml", runtime, ref ErrorDetail);
+        //RunNUnitTests(dir, FRAMEWORK_TESTS, runtime, ref ErrorDetail);
```

Comment:
Run tests with NUnitLite runner. I left the old code commented out so we could switch back easily.